packages/api-explorer/src/components/DocSdkUsage/DocSdkUsage.tsx

```diff
@@ -61,7 +62,8 @@ interface DocSdkUsageProps {
  */
 export const DocSdkUsage: FC<DocSdkUsageProps> = ({ method }) => {
   const examples = useSelector(selectExamplesLode)
-  const sdkLanguage = useSelector(selectSdkLanguage)
+  const alias = useSelector(selectSdkLanguage)
```

Comment:
is this selector still `selectSdkLanguage`, or is it `selectSdkAlias` or just `selectSdk`?